glusterd2/volume/struct.go

```diff
@@ -64,6 +64,7 @@ type Volinfo struct {
 	Transport    string
 	DistCount    int
 	ReplicaCount int
+	ArbiterCount int
```

Comment:
This can be a bool - here and elsewhere below and be named `Arbiter`

Reply:
Same reason as above. I agree that ArbiterCount will always be one (or zero for normal replica volumes)  but it might be better to retain this as an int IMHO.